src/BugsnagUnity/Payload/Error.cs

```diff
@@ -224,6 +243,23 @@ public static Error FromUnityLogMessage(UnityLogMessage logMessage, System.Diagn
             }
         }
 
+        public static bool ShouldSend(System.Exception exception)
+        {
+            if (exception.StackTrace == null)
```

Comment:
Is it possible that an exception has an empty stack trace _and_ it has one of the excluded error classes? it seems possible that an exception matching `ANDROID_JAVA_EXCEPTION_CLASS` could accidentally be sent by this line.

Reply:
That condition was unnecessary, i have removed it 👍